### PR TITLE
Modernize to flexbox

### PR DESCRIPTION
**Pull request by redaxmedia**

### CODE REVIEW

File: templates/default/assets/styles/button.css [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
 /** @section 1. button default */
 
+button
```
Comment (redaxmedia) [inline templates/default/assets/styles/button.css]: Please remove this as this is part of latest harmonize.css ...

File: templates/default/assets/styles/search.css [hunk 2]
```diff
@@ -25,6 +25,7 @@
 .rs-field-search
 {
 	flex-grow: 1;
+	min-width: 0;
```
Comment (redaxmedia) [inline templates/default/assets/styles/search.css]: Please explain why a min-width needed?

Comment (joergsteinhauer) [reply templates/default/assets/styles/search.css]: thats why https://stackoverflow.com/questions/44890719/flex-box-ignoring-width-if-input-in-flexitem

File: templates/default/assets/styles/button.css (hunk 1 above)
Comment (joergsteinhauer) [reply templates/default/assets/styles/button.css]: yep, just forgot to remove...

**Pull request merged**